xcode/Subconscious/Shared/SubconsciousApp.swift

```diff
@@ -677,6 +677,12 @@ struct AppEnvironment {
             data: zettelkastenGrammar
         )
         self.feed.register(name: "zettelkasten", geist: zettelkastenGeist)
+        
+        let comboGeist = try! ComboGeist(
```

Comment:
Note: in future, we should probably not force unwrap, and instead only register geist if it returns some value.

FYI only, no need to change.

Reply:
I took a swing at this, lmk what you think. I'm still learning the languages features of swift so I am likely ignorant of many patterns.